lib/rules/__tests__/index.test.mjs

```diff
@@ -74,6 +76,25 @@ describe('custom message option', () => {
 	);
 });
 
+describe('invalid options', () => {
+	test.each(ruleNames)('"%s" should handle invalid options', async (name) => {
+		const rule = await rules[name];
+		const primaryOption = false;
+		const secondaryOptions = {};
+		const context = {};
+		const root = postcss.root();
+		const result = Object.assign(root.toResult(), {
+			stylelint: { config: { validate: true } },
```

Comment:
note

`validate` configuration property wasn't added
cf #8087

Reply:
`config.validate` is still present. Should we remove it?

https://github.com/stylelint/stylelint/blob/a0c6dc43d32f3c93e7bcef090c06aefd59373e08/lib/utils/validateOptions.mjs#L19-L21